Sources/Core/UI/PreviewViewController.swift

```diff
@@ -12,19 +12,19 @@ import UIKit
 // MARK: - Preview actions
```

Comment:
I used auto-formatting on this file, which is why there are lines that have been deleted and add again.

Reply:
It will be hard to undo. I think the whitespace by the linter is meant to match the indent of the line below.